[CQT-32] Improve syntax definitions (#47)
QuTech-Delft/cQASM-spec: docs/language_specification/tokens/operators_and_punctuators.md

> The lexical representation of cQASM programs includes a number of tokens which serve the function of operators and punctuators:

Reviewer: ```suggestion
The lexical representation of cQASM includes tokens that serve as operators or punctuators.
```
What is meant by 'The lexical representation of cQASM programs'?

Author: I've kept the last part of the suggestion.
